cli_tools_e2e_test/gce_image_import_export/test_suites/import/cli_tests.go

```diff
@@ -273,14 +293,113 @@ func runImageImportWithSubnetWithoutNetworkSpecified(ctx context.Context, testCa
+	runImageImportUEFITest(ctx, testCase, logger, testProjectConfig, testType,
+		"ubuntu-1804", "gs://compute-image-tools-test-resources/uefi/linux-hybrid-ubuntu-1804.vmdk", true, true)
+}
+
+func runImageImportLinuxUEFIMBRTest(ctx context.Context, testCase *junitxml.TestCase, logger *log.Logger,
+	testProjectConfig *testconfig.Project, testType utils.CLITestType) {
+
+	// source created from projects/gce-uefi-images/global/images/ubuntu-1804-bionic-v20200317 and converted from GPT to MBR
+	runImageImportUEFITest(ctx, testCase, logger, testProjectConfig, testType,
+		"ubuntu-1804", "gs://compute-image-tools-test-resources/uefi/linux-ubuntu-mbr-uefi.vmdk", true, true)
+}
+
+func runImageImportWindowsUEFITest(ctx context.Context, testCase *junitxml.TestCase, logger *log.Logger,
+	testProjectConfig *testconfig.Project, testType utils.CLITestType) {
+
+	// source created from projects/gce-uefi-images/global/images/windows-server-2019-dc-core-v20200609
+	runImageImportUEFITest(ctx, testCase, logger, testProjectConfig, testType,
+		"windows-2019", "gs://compute-image-tools-test-resources/uefi/windows-uefi-2019.vmdk", true, true)
+}
+
+func runImageImportWindowsNonUEFITest(ctx context.Context, testCase *junitxml.TestCase, logger *log.Logger,
+	testProjectConfig *testconfig.Project, testType utils.CLITestType) {
+
+	// source created from projects/windows-cloud/global/images/windows-server-2019-dc-v20200114
+	runImageImportUEFITest(ctx, testCase, logger, testProjectConfig, testType,
+		"windows-2019", "gs://compute-image-tools-test-resources/uefi/windows-nonuefi-2019.vmdk", true, false)
+}
+
+func runImageImportUEFITest(ctx context.Context, testCase *junitxml.TestCase, logger *log.Logger,
+	testProjectConfig *testconfig.Project, testType utils.CLITestType, os string, source string, isSourceFile bool,
+	isUEFICompatible bool) {
+
+	suffix := path.RandString(5)
+	imageName := "e2e-test-image-import-" + suffix
+
+	argsMap := map[utils.CLITestType][]string{
+		utils.Wrapper: {"-client_id=e2e", fmt.Sprintf("-project=%v", testProjectConfig.TestProjectID),
+			fmt.Sprintf("-image_name=%v", imageName), fmt.Sprintf("-os=%v", os), getSourceArg(isSourceFile, source),
+			fmt.Sprintf("-zone=%v", testProjectConfig.TestZone), "-inspect",
+		},
+		utils.GcloudProdWrapperLatest: {"compute", "images", "import", imageName, "--quiet",
+			"--docker-image-tag=latest", fmt.Sprintf("--os=%v", os), fmt.Sprintf("--project=%v", testProjectConfig.TestProjectID),
+			getSourceArg(isSourceFile, source),
+			fmt.Sprintf("--zone=%v", testProjectConfig.TestZone), "--inspect",
+		},
+		utils.GcloudLatestWrapperLatest: {"compute", "images", "import", imageName, "--quiet",
+			"--docker-image-tag=latest", fmt.Sprintf("--os=%v", os), fmt.Sprintf("--project=%v", testProjectConfig.TestProjectID),
+			getSourceArg(isSourceFile, source),
+			fmt.Sprintf("--zone=%v", testProjectConfig.TestZone), "--inspect",
```

Comment:
Is there `--inspect` flag in gcloud?

Reply:
I wasn't planning on that being added to gcloud